### PR TITLE
Feature Database Download

### PR DESCRIPTION
**Pull request by donpaul999**
- Add Database Download
- Messages are load from database
- Message are available for users on index page after "Get Messages" button is clicked

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -42,14 +42,14 @@ $('.carousel').carousel({
 
 function getMessagesFromServer() {
   fetch('/data').then(response => response.json()).then((messages) => {
+    console.log(messages);
```
Comment (ramaroberto) [inline portfolio/src/main/webapp/script.js]: Remove this line, there shouldn't be any console.log (s) in the final version.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -30,15 +33,20 @@
...
-  private ArrayList<String> messages;
-
-  @Override
-  public void init(){
-    messages = new ArrayList<String>();
-  }  
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
```
Comment (ramaroberto) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Remove this blank line

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -30,15 +33,20 @@
...
 
-  private ArrayList<String> messages;
-
-  @Override
-  public void init(){
-    messages = new ArrayList<String>();
-  }  
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (ramaroberto) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:40]: Would you mind adding a comment in a Javadoc style to explain what this function is doing? All public functions should have documentation.

http://www.drjava.org/docs/user/ch10.html

**Review by ramaroberto — CHANGES_REQUESTED** [review]
Same as before: Feel free to ignore the comments if you're changing that same piece of code in a following PR; but please ACK them by telling me this.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (donpaul999) [reply portfolio/src/main/webapp/script.js]: Solved

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (donpaul999) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:40]: Solved

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -30,26 +33,36 @@
...
 
-  private ArrayList<String> messages;
-
-  @Override
-  public void init(){
-    messages = new ArrayList<String>();
-  }  
-
+  /**
+  * Loads messages from database and transform them into a JSON object
```
Comment (ramaroberto) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: transforms*

**Review by ramaroberto — CHANGES_REQUESTED** [review]
Feel free to merge after solving this small nit.

**Pull request merged**